ci-operator/step-registry/sandboxed-containers-operator/install-trustee-operator/sandboxed-containers-operator-install-trustee-operator-commands.sh

```
@@ -0,0 +1,702 @@
+      echo ">>> ⚠ No resource GET requests found"
+    fi
+
+  else
+    echo ">>> WARNING: Could not find KBS pod in namespace ${TRUSTEE_NAMESPACE}"
+    echo ">>> Listing all pods in namespace:"
+    oc get pods -n "${TRUSTEE_NAMESPACE}" || true
+  fi
+
+  # Clean up the pod
+  echo ">>> Cleaning up kbs-client pod"
+  oc delete pod/${kbs_client_pod} -n ${kbs_client_namespace} --ignore-not-found=true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)